cnames_active.js

```diff
@@ -1980,6 +1980,7 @@ var cnames_active = {
   "webpack": "webpack.github.io/webpack.js.org",
   "webpack-config-plugins": "namics.github.io/webpack-config-plugins",
   "webpack-inspect": "alexmost.github.io/webpack-inspect",
+  "wechaty": "hosting.gitbook.com", // We are wechaty.github.io/wechaty.js.org
```

Comment:
```suggestion
  "wechaty": "hosting.gitbook.com",
```
Please remove the comment :)